ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -183,3 +185,14 @@ def map_resource(geocat_resource, issued, modified, rights):
         resource_dict['download_url'] = ""
     resource_dict['language'] = geocat_resource['language']
     return resource_dict
+
+
+def map_service(geocat_service, issued, modified, description, rights):
+    resource_dict = {}
+    resource_dict['description'] = description
+    resource_dict['issued'] = issued
+    resource_dict['modified'] = modified
+    resource_dict['rights'] = rights
+    resource_dict['media_type'] = geocat_service.get('media_type', '')
+    resource_dict['url'] = geocat_service.get('url', '')
```

Comment:
It would be cleaner imo to create this as a dict literal:
```suggestion
    resource_dict = {
      'description': description,
      'issued': issued,
      'modified': modified,
      // etc
    }
```